README.md

```diff
@@ -62,6 +62,23 @@ After changing this file, you need to run
 
 so that the changes get transferred to the initramfs.
 
+Enable yubikey-luks initramfs module
+-------------------------
+
+In order to use yubikey-luks for unlocking LUKS encrypted volume at boot you may append keyscript=/usr/share/yubikey-luks/ykluks-keyscript to the /etc/crypttab file. Example:
```

Comment:
"may" is a bit week here.
I would recommend "must".
Down below is the alternative.

However, I realize, that this is a bit tricky - when I was testing this, I actually missed this point and was wondering, why it did not work out.
I think we somehow need to either add this to the postinst of the debian package or we should output a hint, after running ``yubikey-luks-enroll``.

I also see, that the text from README.md is actually missing in the debian package.
We should put it in the man page. What do you think?

Reply:
We can put it in manpage (which would need to rewrite it for man format) or install as is in `/usr/share/doc/yubikey-luks`.